examples/born/apbs-mol-fem-extmesh.in

```diff
@@ -3,13 +3,14 @@
 #############################################################################
 read 
 	mol pqr ion.pqr
-	mesh mcsf mesh.m
+	# WARN: The following no longer works
+	# mesh mcsf mesh.m
 end
 elec name born
 	fe-manual
 	domainLength 12 12 12
 	etol 0.1
-	ekey glob
+	ekey global
```

Comment:
Is the backwards-compatible syntax supported as well?

Reply:
I used the [fe-manual](https://apbs.readthedocs.io/en/latest/using/input/elec/fe-manual.html) specification. I might be able to add an issue to implement a "re-write" grammar to check for `ekey glob` and change it to `ekey global` to support backwards compatibility. Would you rather support backwards compatibility or depricate the use of `glob`?

Reply:
Ha!  I can't agree with my own documentation... 😂  Let's leave it the way you have it and see if anything breaks.  The example runs successfully, right?

Reply:
I created a function to check for deprecated values for keys. It works for this example so I am reverting all files with `ekey glob` back to the way they were. The code produces a meaningful WARNING about where the deprecated value is located, makes the substitution internally, and continues on. Right now it only scales to values of keys. It does not scale to invalid keys or other keywords (e.g. `geoflow` to `geoflow-auto`.